server/mscalendar/availability.go

```diff
@@ -215,22 +219,22 @@ func (m *mscalendar) setStatusFromCalendarView(user *store.User, currentStatus s
 
 	if len(user.ActiveEvents) == 0 {
 		var err error
-		if currentStatus == model.STATUS_DND {
+		if currentStatus == busyStatus {
```

Comment:
Hmm if I have my busy preference set to `Away`, and I'm currently set to `DND`, it will change me to `Away`. I'm not entirely sure this makes sense, because I'm already set to the busiest of busy.

@asaadmahmood For context, the user needs to be able to choose if they want to receive notifications while being marked "In a meeting". We are doing this by letting them choose between `DND` or `Away` in their plugin preferences.

Reply:
As discussed through zoom, if the user want to be away during meetings, it will change to away.